netebpfext/net_ebpf_ext.c

```diff
@@ -48,11 +48,48 @@ static ebpf_ext_attach_hook_provider_registration_t* _ebpf_xdp_hook_provider_reg
 static ebpf_ext_attach_hook_provider_registration_t* _ebpf_bind_hook_provider_registration = NULL;
 static ebpf_extension_provider_t* _ebpf_xdp_program_information_provider = NULL;
 static ebpf_extension_provider_t* _ebpf_bind_program_information_provider = NULL;
-static ebpf_extension_data_t* _ebpf_xdp_program_information_provider_data = NULL;
-static ebpf_extension_data_t* _ebpf_bind_program_information_provider_data = NULL;
 
 #define RTL_COUNT_OF(arr) (sizeof(arr) / sizeof(arr[0]))
```

Comment:
Line 42 includes ebpf_platform.h which defines EBPF_COUNT_OF, so RTL_COUNT_OF should be removed from this file.  Ok to do in a separate PR if you prefer, since it was bad before this PR.

Reply:
Will do so in a separate PR.